tests/rptest/tests/workload_upgrade_config_defaults.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2023 Redpanda Data, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+from rptest.services.admin import Admin
+from rptest.services.workload_protocol import PWorkload
+
+PROP_NAME = 'log_segment_ms_min'
+PROP_NEW_VALUE = 9999999
```

Comment:
I'm a bit confused by this value. Shouldn't this be set to something under 10 minutes? I thought the idea was to ensure that clusters with values set too low before the default change are unaffected